GermanFilter/sections/specific.txt

```diff
@@ -2123,7 +2123,7 @@ ruhr24.de,wetterauer-zeitung.de,giessener-allgemeine.de,fr.de,wlz-online.de,da-i
 wallstreet-online.de##body > #siteteaser
 wallstreet-online.de##.newsFlagSlider
 wallstreet-online.de###mooTickerContainer
-! https://github.com/AdguardTeam/AdguardFilters/pull/6023*
+! https://github.com/AdguardTeam/AdguardFilters/pull/60233
```

Comment:
No need to include link to pull request.

Reply:
Has been removed now.